index.html

```diff
@@ -43,5 +45,195 @@ <h1>Hey There. I'm Carlyle</h1>
       </ul>
     </section>
   </header>
+  <section class="works">
+    <div class="title">
+      <h2 class="titlewords">My Recent Works</h2>
+      <span class="line"></span>
+    </div>
+    <div class="intro">
+      <div class="intimage">
+        <img src="images/projectimg.png" alt="works intro image placeholder">
+      </div>
```

Comment:
- [ ] [Required] Sticking to design templates or mockups is a good professional attribute of a frontend developer. Your project looking exactly as the design template increases your chances of getting hired easily. **Use the margins, paddings, colors, etc provided in the Figma template.** Let's rework this.
## Your Design
![image](https://user-images.githubusercontent.com/45668340/148661717-fb8da25e-c868-40c6-bbe2-9115c06e7f7f.png)
## The Figma's
![image](https://user-images.githubusercontent.com/45668340/148661733-e40d66a6-7c89-4818-b5da-c982bfb005ef.png)